### DISCUSSION TITLE
Cannot access the documentation web pages

### DISCUSSION THREAD

**Discussion by mellexpro** [datalust/seq-tickets#1319] [Help]
I am trying to learn how to do filtering but I can't access any documentation web page other than the what' new site page, every click of a link in the menu bar brings the browser (I tried multiple different browser types) to the what's new page again.

**Comment by nblumhardt** [comment]
Hi @mellexpro - sorry to hear about the difficulties! Thanks for letting us know.

I've just checked over the docs and can't reproduce this now, with Firefox, Chrome, or Edge.

My best guess is that this was a bad deploy, and has subsequently been fixed (we use a third-party service for doc hosting), so hopefully you'll be able to get to the docs you need, now.

The best place to start with filtering is: https://docs.datalust.co/docs/the-seq-query-language

If not, would it be possible to share some screenshots (either here or via our `support@datalust.co` address) and to check the Developer Tools console in your browser for any relevant errors/information?

Best regards,
Nick

**Reply by mellexpro** [reply]
@nblumhardt 
Thanks for your response. The documentation is working again, thanks!
It might have been a bad deployment, just happened to coincide with my need to learn some queries!
